src/components/dashboard/sidebar/tests/DashboardSidebar.test.jsx

```diff
@@ -101,11 +100,18 @@ describe('<DashboardSidebar />', () => {
     );
     expect(screen.queryByText(SUBSCRIPTION_SUMMARY_CARD_TITLE)).toBeFalsy();
   });
-  test('subscription summary card is not displayed when enterprise subscription is available but user subscription is not available', () => {
+  test('subscription summary card is not displayed when enterprise subscription is available but user license is not available', () => {
     renderWithRouter(
       <DashboardSidebarContext
-        initialAppState={appStateWithSubscription}
-        initialUserSubsidyState={defaultUserSubsidyState}
+        initialAppState={initialAppState}
+        initialUserSubsidyState={{
+          ...defaultUserSubsidyState,
+          subscriptionPlan: {
+            daysUntilExpiration: 70,
+            isActive: true,
+            expirationDate: '2021-10-25',
```

Comment:
Will this cause a test to fail on Oct. 26?  I'm guessing the logic is based only on `daysUntilExpiration`, but wanted to double-check.

Reply:
The logic for displaying the subscription summary card isn't based on `expirationDate`; it shows for active/expiring/expired subscriptions, but its display is conditional on the presence of a user license.

The date shouldn't have any bearing on this particular test.